WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -1063,14 +1061,18 @@ function(params) {
 			id:						this._msgBodyDivId,
 			hidden:					true,
 			html:					"<div>" + (this._cleanedHtml || html) + "</div>",
-			styles:					inner_styles,
 			noscroll:				!this._scrollWithIframe,
 			posStyle:				DwtControl.STATIC_STYLE,
 			processHtmlCallback:	callback,
 			useKbMgmt:				true,
 			title:                  this._getIframeTitle()
 		};
 
+		// bug fix #9475 - IE isnt resolving MsgBody class in iframe so set styles explicitly
+		if(AjxEnv.isIE) {
```

Comment:
I prefer having a space after if condition
let me know if that drastically affects code readability